src/web/app/pages-instructor/instructor-student-activity-logs/instructor-student-activity-logs.component.ts

```diff
@@ -0,0 +1,367 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { NgbDateParserFormatter } from '@ng-bootstrap/ng-bootstrap';
+import { finalize } from 'rxjs/operators';
+import { CourseService } from '../../../services/course.service';
+import { FeedbackSessionsService } from '../../../services/feedback-sessions.service';
+import { LogService } from '../../../services/log.service';
+import { StatusMessageService } from '../../../services/status-message.service';
+import { StudentService } from '../../../services/student.service';
+import { TimezoneService } from '../../../services/timezone.service';
+import { ApiConst } from '../../../types/api-const';
+import {
+  Course, FeedbackSession,
+  FeedbackSessionLog, FeedbackSessionLogEntry,
+  FeedbackSessionLogs, FeedbackSessions,
+  Student,
+} from '../../../types/api-output';
+import { SortBy } from '../../../types/sort-properties';
+import { DatePickerFormatter } from '../../components/datepicker/datepicker-formatter';
+import { DateFormat } from '../../components/datepicker/datepicker.component';
+import { ColumnData, SortableTableCellData } from '../../components/sortable-table/sortable-table.component';
+import { TimeFormat } from '../../components/timepicker/timepicker.component';
+import { ErrorMessageOutput } from '../../error-message-output';
+
+/**
+ * Model for searching of logs
+ */
+interface SearchLogsFormModel {
+  logsDateFrom: DateFormat;
+  logsDateTo: DateFormat;
+  logsTimeFrom: TimeFormat;
+  logsTimeTo: TimeFormat;
+  logType: string;
+  feedbackSessionName: string;
+  studentEmail: string;
+  showActions: boolean;
+  showInactions: boolean;
+}
+
+interface LogType {
+  label: string;
+  value: string;
+}
+
+/**
+ * Model for displaying of feedback session logs
+ */
+interface FeedbackSessionLogModel {
+  publishedDate: string;
+  feedbackSessionName: string;
+  logColumnsData: ColumnData[];
+  logRowsData: SortableTableCellData[][];
+  isTabExpanded: boolean;
+}
+
+/**
+ * Component for instructor logs
+ */
+@Component({
+  selector: 'tm-instructor-audit-logs-page',
```

Comment:
The selector is incorrect. Although it doesn't really matter, let's not make a habit out of this.